src/main/java/BadApple/task/Storage.java

```diff
@@ -0,0 +1,161 @@
+package BadApple.task;
+
+import BadApple.main.BadAppleException;
+
+import java.io.*;
```

Comment:
I don't think wildcard import is recommended. Perhaps list the imported classes explicitly?